nix/pkgs/default.nix

```diff
@@ -96,6 +96,13 @@ let
   # not available in 20.03 and we depend on several recent changes
   # including stylish-haskell support
   nix-pre-commit-hooks = import (sources."pre-commit-hooks.nix");
+  #nix-pre-commit-hooks = import ../../../../cachix/pre-commit-hooks.nix/default.nix;
```

Comment:
Commented-out-code. BTW you know you can override niv sources with an environment variable now?

Reply:
Yikes, thanks for noticing that one. My comments leave a trail of my hacking around :sweat_smile: 
Also: no, i totally did not know that I  can override niv sources with an environment variable! That sounds both useful, and absolutely terrifying at the same time ;)